src/main/java/nextstep/subway/line/application/LineService.java

```diff
@@ -40,33 +42,37 @@ public LineResponse saveLine(LineRequest request) {
     public List<LineResponse> findLines() {
         List<Line> persistLines = lineRepository.findAll();
         return persistLines.stream()
-                .map(line -> {
-                    List<StationResponse> stations = line.getStations()
-                            .stream()
-                            .map(it -> StationResponse.of(it))
-                            .collect(Collectors.toList());
-                    return LineResponse.of(line, stations);
-                })
-                .collect(Collectors.toList());
+            .map(line -> {
+                List<StationResponse> stations = line.getStations()
+                    .stream()
+                    .map(it -> StationResponse.of(it))
+                    .collect(Collectors.toList());
```

Comment:
자바16부터는 stream 최종연산으로 toList()가 추가되었더라구요 
다만 Collections.toList() 와는 달리 unmodifiableList를 반환해주더라구요 
이미 아실수도 있겠지만 참고만 부탁드립니다 
```suggestion
                    .toList();
```